README.md

```diff
@@ -106,40 +106,104 @@ See [parse feature](features/parse.feature) for the details on using the `drafte
 ## Build
 1. Clone the repo + fetch the submodules:
 
-	```sh
-	$ git clone --recursive git://github.com/apiaryio/drafter.git
-	$ cd drafter
-	```
+    ```sh
+    $ git clone --recursive git://github.com/apiaryio/drafter.git
+    $ cd drafter
+    ```
 
 2. Build & test Drafter:
 
-	```sh
-	$ ./configure
-	$ make test
-	```
+    ```sh
+    $ ./configure
+    $ make test
+    ```
 
-	To include integration tests (using Cucumber) use the `--include-integration-tests` flag: 
+    To include integration tests (using Cucumber) use the `--include-integration-tests` flag:
+
+    ```sh
+    $ ./configure --include-integration-tests
+    $ make test
+    ```
 
-	```sh
-	$ ./configure --include-integration-tests
-	$ make test
-	```
-	
 We love **Windows** too! Please refer to [Building on Windows](https://github.com/apiaryio/drafter/wiki/Building-on-Windows).
-		
+
 ### Drafter command line tool
 1. Build `drafter`:
-	
-	```sh
-	$ make drafter
-	```
+
+    ```sh
+    $ make drafter
+    ```
 
 2. Install & use `drafter`:
 
-	```sh
-	$ sudo make install
-	$ drafter --help
-	```
+    ```sh
+    $ sudo make install
+    $ drafter --help
+    ```
+
+## drafter.js
```

Comment:
I think we should move this whole section to be after `# Bindings` because we want people to use native bindings first and then if they are not working go to already compiled versions.